package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "version": "3.2.4",
+  "description": "Add-on to jQuery Ajax to enable unobtrusive options in data-* attributes",
+  "main": "jquery.unobtrusive-ajax.js",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/aspnet/jquery-ajax-unobtrusive"
+  },
+  "keywords": [
+    "unobtrusive",
+    "ajax",
+    "mvc",
+    "asp.net",
+    "jquery"
+  ],
+  "author": "Microsoft",
+  "license": "http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm",
```

Comment:
Looks like this URL auto-redirects to HTTPS anyway, so we should absolutely just use that.